opennms-doc/guide-admin/src/asciidoc/text/compatibility.adoc

```diff
@@ -0,0 +1,18 @@
+[[compatibility]]
+== Compatibility
+
+{opennms-product-name} {opennms-version} requires the following component versions:
+
+[options="header, autowidth"]
+|===
+| Component               | Version Compatibility
+| OpenNMS Integration API | 0.2.1
+| Java Development Kit    | OpenJDK 8, OpenJDK 11
+| PostgreSQL              | 10.x and 11.x
+| Cassandra               | 3.11+
+| Newts                   | 1.5.2
+| Helm                    | 3+
+| Kafka                   | 2.3.0
+| Elasticsearch           | 6.2.4 until 7.3.0
```

Comment:
Pretty sure we don't support Elasticsearch 6.x in Horizon 25.

Reply:
What I know here so far, Elasticsearch version dependency is driven by our elasticsearch-drift-plugin which is installed on the Elasticsearch node. The first version was 1.0.0 which pointed to Elasticsearch 6.2.4 which is the reason I've picked it :) Now is the best chance to clarify this :)

Reply:
H25 -> Only ES 7.x: http://docs.opennms.org/opennms/releases/25.0.0/releasenotes/releasenotes.html#_elasticsearch_7_x_support